tests/unit/protocols/input/rest-xml.json

```diff
@@ -73,16 +73,16 @@
       {
         "given": {
           "http": {
-            "method": "PUT",
+            "method": "GET",
```

Comment:
@jamesls 
I was the one to originally added this test case. I added it to help improve coverage on our serializers. The PUT method was just used because of me just copying and pasting from previous examples. The test case was to cover if there was no input shape for the operation and so I believe the expected value is incorrect so it needed changing. I also cannot think of an example of a ``PUT`` request with no input parameters.